src/TNBase.DataStorage/Service/ServiceLayer.cs

```diff
@@ -65,36 +65,40 @@ public List<Listener> GetOnlineOnlyListenersOrderedBySurname()
             return context.Listeners.Where(x => !x.Status.Equals(ListenerStates.DELETED) && x.OnlineOnly).OrderBy(x => x.Surname).ToList();
         }
 
-        public List<Listener> GetNextWeekBirthdays()
```

Comment:
There are some tests failing do to this. Also, you will need to write some tests for the new method. Are you familiar with that? We can discuss it tomorrow.